src/ontoweaver/tabular.py

```diff
@@ -594,6 +610,22 @@ def __call__(self):
             columns=subject_columns, transformer_type=subject_transformer_class,
             node_type=source_t, properties=properties_of.get(subject_type, {}), **subject_kwargs)
 
+        # Extract metadata for the source type.
+        if metadata_list:
+            if subject_type:
+                metadata.setdefault(subject_type, {})
+                for item in metadata_list:
+                    metadata[subject_type].update(item)
+                for key in k_metadata_column_placeholder:
+                    if key in metadata[subject_type]:
+                    # Use the value of k_metadata_column_placeholder as the key.
+                        key_name = metadata[subject_type][key]
+                        # Remove the k_metadata_column_placeholder key from the metadata dictionary.
+                        del metadata[subject_type][key]
+                        if subject_columns:
```

Comment:
We should assert here that `key_name` does not already exist in the dictionary, or we are going to overwrite an existing property by the user. Maybe this can be double-checked at the parsing stage as well.